client/blocks/reader-combined-card/post.jsx

```diff
@@ -0,0 +1,135 @@
+import ReaderFeaturedVideo from 'blocks/reader-featured-video';
+import * as stats from 'reader/stats';
+
+class ReaderCombinedCardPost extends React.Component {
+	static propTypes = {
+		post: React.PropTypes.object.isRequired,
+		streamUrl: React.PropTypes.string,
+		onClick: React.PropTypes.func,
+	};
+
+	propagateCardClick = () => {
+		// If we have an discover pick post available, send the discover pick to the full post view
+		const postToOpen = get( this.props, 'discoverPick.post' ) || this.props.post;
+		this.props.onClick( postToOpen );
+	}
+
+	handleCardClick = ( event ) => {
+		const rootNode = ReactDom.findDOMNode( this ),
+			selection = window.getSelection && window.getSelection();
+
+		// if the click has modifier or was not primary, ignore it
+		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
+			if ( closest( event.target, '.reader-combined-card__post-title-link', true, rootNode ) ) {
+				stats.recordPermalinkClick( 'card_title_with_modifier', this.props.post );
+			}
+			return;
+		}
+
+		if ( closest( event.target, '.should-scroll', true, rootNode ) ) {
+			setTimeout( function() {
+				window.scrollTo( 0, 0 );
+			}, 100 );
+		}
+
+		// declarative ignore
+		if ( closest( event.target, '.ignore-click, [rel~=external]', true, rootNode ) ) {
+			return;
+		}
+
+		// ignore clicks on anchors inside inline content
+		if ( closest( event.target, 'a', true, rootNode ) && closest( event.target, '.reader-excerpt', true, rootNode ) ) {
+			return;
+		}
+
+		// ignore clicks when highlighting text
+		if ( selection && selection.toString() ) {
+			return;
+		}
+
+		// programattic ignore
+		if ( ! event.defaultPrevented ) { // some child handled it
+			event.preventDefault();
+			this.propagateCardClick();
+		}
+	}
+
+	render() {
+		const { post, streamUrl } = this.props;
+		const hasAuthorName = has( post, 'author.name' );
+
+		let featuredAsset = null;
+		if ( post.canonical_media && post.canonical_media.mediaType === 'video' ) {
+			featuredAsset = <ReaderFeaturedVideo { ...post.canonical_media } videoEmbed={ post.canonical_media } allowPlaying={ false } />;
+		} else if ( post.canonical_media ) {
+			featuredAsset = <ReaderFeaturedImage imageWidth={ 100 } imageUrl={ post.canonical_media.src } href={ post.URL } />;
+		}
+
+		const recordDateClick = () => {
+			recordPermalinkClick( 'timestamp_combined_card', post );
+		};
+
+		return (
+			<li className="reader-combined-card__post" onClick={ this.handleCardClick }>
+				<div className="reader-combined-card__featured-image-wrapper">
+					{ featuredAsset }
+				</div>
+				<div className="reader-combined-card__post-details">
+					<AutoDirection>
+						<h1 className="reader-combined-card__post-title">
+							<a className="reader-combined-card__post-title-link" href={ post.URL }>{ post.title }</a>
+						</h1>
+					</AutoDirection>
+					<ReaderExcerpt post={ post } isDiscover={ false } />
```

Comment:
Nope - good spot! The combined card should probably accept this as a prop and pass it down to the post. I'll fix.